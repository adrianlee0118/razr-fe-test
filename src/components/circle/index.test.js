import React from "react";
import ReactDOM from "react-dom";
import renderer from "react-test-renderer";
import Circle from "./index";

describe("Circle", () => {
  //Component test
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<Circle />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  //Snapshot test
  test("has a valid snapshot", () => {
    const component = renderer.create(<Circle />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
