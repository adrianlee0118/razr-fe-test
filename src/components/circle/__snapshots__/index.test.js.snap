// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Circle has a valid snapshot 1`] = `
<div
  style={
    Object {
      "backgroundColor": "blue",
      "borderRadius": "50%",
      "display": "inline-block",
      "height": NaN,
      "margin": 20,
      "padding": 10,
      "width": NaN,
    }
  }
/>
`;
